mlflow/R/mlflow/R/project-run.R

```diff
@@ -8,23 +8,20 @@
 #' @param param_list A list of parameters.
 #' @param experiment_id ID of the experiment under which to launch the run.
 #' @param experiment_name Name of the experiment under which to launch the run.
-#' @param mode Execution mode to use for run.
-#' @param cluster_spec Path to JSON file describing the cluster to use when launching a run on Databricks.
-#' @param git_username Username for HTTP(S) Git authentication.
-#' @param git_password Password for HTTP(S) Git authentication.
+#' @param backend Execution backend to use for run.
+#' @param backend_spec Path to JSON file describing the cluster to use when launching a run on Databricks.
```

Comment:
Maybe "backend_config" instead, and comment to "Path to JSON file which will be passed as config to the backend. For Databricks, this should be a cluster spec."